src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClient.kt

```diff
@@ -61,13 +60,13 @@ class ActivitiesAppointmentsClient(private val activitiesAppointmentsApiWebClien
           tierCode = AppointmentSeriesCreateRequest.TierCode.TIER_1,
           inCell = false,
           startDate = startDate,
-          startTime = startTime.toIsoTime(),
-          endTime = endTime.toIsoTime(),
+          startTime = startTime.toHourMinuteStyle(),
+          endTime = endTime.toHourMinuteStyle(),
           internalLocationId = internalLocationId,
+          organiserCode = AppointmentSeriesCreateRequest.OrganiserCode.OTHER,
```

Comment:
Also thought it prudent to specify the organiser.  Other choices are PRISON_STAFF, PRISONER and EXTERNAL_PROVIDER.  Wasn't sure if EXTERNAL_PROVIDER would be a better choice.